README.md

```diff
@@ -3,17 +3,30 @@
 [![Build Status](https://travis-ci.org/solid/solid-auth-client.svg?branch=master)](https://travis-ci.org/solid/solid-auth-client)
 [![Coverage Status](https://coveralls.io/repos/github/solid/solid-auth-client/badge.svg?branch=master)](https://coveralls.io/github/solid/solid-auth-client?branch=master)
 
-Opaquely authenticates solid clients
+Opaquely authenticates [Solid](https://github.com/solid/) clients
 
 ## About
 
+### What is this?
+
 Solid currently supports two cross-origin authentication protocols,
 [WebID-TLS](https://www.w3.org/2005/Incubator/webid/spec/tls/) and
 [WebID-OIDC](https://github.com/solid/webid-oidc-spec).
 
 This library abstracts away the implementation details of these specs so that
 clients don't have to handle different authentication protocols.
 
+### Why might I need this?
+
+If you're building a web app and want to identify users with Solid, or store
+personal information on your user's solid account, you'll have to authenticate
+them.  This library provides a simple API for logging in, logging out, and
+fetching resources with authenticated credentials.
+
+### How can I use this?
+
+The simplest way to use this library is to install it via `npm` or `yarn`.  You can then use the ES6 module (`import { login, currentUser, logout } from 'solid-auth-client'`), or you can grab the transpiled UMD bundle from `node_modules/solid-auth-client/dist-lib/solid-auth-client.bundle.js`.
+
 ## API
```

Comment:
Takes a bit of time here until we get to the meat. Could it make sense to put the types at the end?